static/downloadables/runPrsCLI.sh

```diff
@@ -280,27 +297,22 @@ learnAboutParameters () {
                 echo "If the output file is in JSON format, the results will, by default, be in verbose format."
                 echo -e "${LIGHTRED}**NOTE:${NC} There is no condensed version of JSON output."
                 echo "" ;;
-            11 ) echo -e "${MYSTERYCOLOR} -g sex dependent associations: ${NC}"
+            12 ) echo -e "${MYSTERYCOLOR} -g sex dependent associations: ${NC}" # fix wording
                 echo "Though a rare occurence, some studies have duplicates of the same SNP that differ by which"
```

Comment:
Will this do?
A rare handful of studies report beta values or odds ratios dependent on biological sex. The system default is to exclude sex dependent SNPs from calculations. You can include sex dependent associations by selecting male (M) and/or female (F).

Reply:
Not quite, we don't exclude them anymore, they are just used in a separate calculation that only does male or only does female. So if a study reports 10 snps, 3 associated with females, 4 associated with males, and the rest with no sex association, we would be returning 3 scores. One using the snps associated with females, one using the snps associated with males, and one with the snps not associated with sex. I think now, if they chose one or the other we only look at studies that have that in their metadata. I'm not 100% sure

Reply:
I updated the wording, check it out and let me know what you think.